vpr/src/util/mmap_file.h

```diff
@@ -0,0 +1,18 @@
+#ifndef MMAP_FILE_H_
+#define MMAP_FILE_H_
```

Comment:
Since it looks like this is potentially re-usable code, perhaps this should be in libvtrutil?